awx/main/tasks.py

```diff
@@ -722,12 +722,12 @@ def build_private_data_files(self, instance, **kwargs):
             ssh_ver = get_ssh_version()
             ssh_too_old = True if ssh_ver == "unknown" else Version(ssh_ver) < Version("6.0")
             openssh_keys_supported = ssh_ver != "unknown" and Version(ssh_ver) >= Version("6.5")
-            for credential, data in private_data.get('credentials', {}).iteritems():
+            for credential, data in list(private_data.get('credentials', {}).items()):
```

Comment:
This is casting to list but `items` is being called. `list` object does not have the `items` method.